stracker/stracker/settings.py

```diff
@@ -132,3 +135,7 @@
 # https://docs.djangoproject.com/en/3.2/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+AUTH_USER_MODEL = "customers.Customer"
+
+LOCALE_PATHS = (str(BASE_DIR / "locale/"),)
```

Comment:
```suggestion
LOCALE_PATHS = (BASE_DIR / "locale/",)
```
str'ye gerek olmadan çalışıyor.

Reply:
Teşekkürler düzeltme için.